packages/odyssey-react-mui/src/FieldError.tsx

```diff
@@ -14,24 +14,36 @@ import { memo } from "react";
 
 import { FormHelperText } from "@mui/material";
 import { ScreenReaderText } from "./ScreenReaderText";
+import { ErrorMessagesList } from "./ErrorMessagesList";
 import { useTranslation } from "react-i18next";
 
 import type { AllowedProps } from "./AllowedProps";
+import { Box } from "./Box";
 
 export type FieldErrorProps = {
   id?: string;
-  text: string;
+  message?: string;
+  messagesList?: string[];
 } & AllowedProps;
 
-const FieldError = ({ id, testId, text, translate }: FieldErrorProps) => {
+const FieldError = ({
+  id,
+  message,
+  messagesList,
+  testId,
+  translate,
+}: FieldErrorProps) => {
   const { t } = useTranslation();
 
   return (
     <FormHelperText data-se={testId} error id={id} translate={translate}>
       <ScreenReaderText translate={translate}>{`${t(
         "fielderror.screenreader.text"
       )}:`}</ScreenReaderText>
-      {text}
+      <Box>
+        {message && message}
```

Comment:
```suggestion
        {message}
```
`false`, `null`, and `undefined` will not render anyway.